dbms/include/DB/Functions/FunctionsReinterpret.h

```diff
@@ -158,8 +158,7 @@ class FunctionReinterpretStringAs : public IFunction
 			for (size_t i = 0; i < size; ++i)
 			{
 				ToFieldType value = 0;
-				memcpySmallAllowReadWriteOverflow15(
-					&value, &data_from[offset], std::min(sizeof(ToFieldType), offsets_from[i] - offset - 1));
+				memcpy(&value, &data_from[offset], std::min(sizeof(ToFieldType), offsets_from[i] - offset - 1));
```

Comment:
I have checked only superficially but these seem to be the only two places where memcpySmall is used to copy to integer values on the stack.